app/models/intervention_output.rb

```diff
@@ -93,6 +93,15 @@ class InterventionOutput < InterventionProductParameter
     end
   end
 
+  def choose_default_name(output)
+    if output.name.eql?("blank")
+      land_parcel_name = self.group.targets.first.product.name rescue ""
+      culture_name = self.variant.name rescue ""
```

Comment:
Could you replace the `rescue`s by conditionals? It is more explicit and will facilitate debug later on.